docs/UserGuide.md

```diff
@@ -296,19 +296,24 @@ Refer to [properties of contact](#properties-of-contact) on the accepted formats
 
 ###  Update contact's photo: `updatephoto`
 
-Updates the photo of an existing contact.
+Besides the ability to edit a contact's information, CampusConnect also provides you with the freedom to customise how it looks in your address book.
```

Comment:
let us standardize the nouns we use! `address book` should be replaced. also, let us consider the value proposition of your feature as well for the user. (updatephoto does not seem like a customization)

perhaps consider the following

```suggestion
If you are afraid that you may forget who the saved contact is in CampusConnect, our application enables you to update the photo of your contact so that you can visually remember and recognize them!
```